### ISSUE TITLE
Can't see any entry on any folder, using any account

### ISSUE THREAD

**Issue by L30Bola** [nilsteampassnet/TeamPass#1882]
### Steps to reproduce
1. Login as some non-admin user
2. Create or just try to view some entry

### Expected behaviour
Should be able to view the entry.

### Actual behaviour
Can't see any entry, but it's possible to search for the entry.

### Server configuration
**Operating system**:
CentOS 7.3.1611 

**Docker Version**:
17.06-ce

**Container Version**:
teampass/teampass:latest

Don't know if it's relevant but the MySQL Server is another Docker container. It's using the latest 5.6 version available to a CentOS, which is 5.6.37.

### Client configuration
**Browser:**
Google Chrome Version 60.0.3112.113 (Official Build) (64-bit)
Mozilla Firefox 55.0.2 (64-bit)

**Operating system:**
Fedora 26

**Comment by nilsteampassnet** [comment]
Has the user the role permitting to view those entries?

**Comment by L30Bola** [comment]
I tried with two users, both had permissions to view and edit the entries. I could even create a new entry and any user wouldn't see the entry, but the entry would appear if it was searched for.

**Comment by L30Bola** [comment]
I made a kinda-short video to try to show the bug (It's not listed): https://youtu.be/9auUGNNyfoM

**Comment by nilsteampassnet** [comment]
Thank you for the video.
Do you have any errors in the log?

**Comment by L30Bola** [comment]
I've seen the nginx config inside the container and it logs everything to stdout, so I'm attaching the Docker log here. 
[teampass-log.txt](https://github.com/nilsteampassnet/TeamPass/files/1257443/teampass-log.txt)

**Comment by nilsteampassnet** [comment]
This seems to be the `access.log` file. It doesn't trace the errors, just the access operations.
Can you attach the `php error log` one?

**Comment by L30Bola** [comment]
Well, I'm trying to enable the error logging for PHP inside the container, since it's disabled by default.
![image](https://user-images.githubusercontent.com/10711497/29821837-b2413e68-8c9f-11e7-918a-eacb0bafe1a6.png)

I didn't find the `php.ini`, but I found a bunch of `.ini` files that appear to alter the PHP variables. So I edited it to add the following lines:
```
display_error = 1
error_log = /var/log/php/php-error.log
error_reporting = E_ALL
log_errors = 1
log_errors_max_len = 0
```
Then, It appeared to work, but the log file isn't being written. I tried to access some pages so some errors should be logged. Still no errors logged. So I restarted the TeamPass application container and now the interface shows me the following blank message:
![image](https://user-images.githubusercontent.com/10711497/29822381-a31b3d38-8ca1-11e7-8973-f96e5ddaf24b.png)

After some little debugging I found that the `error_reporting = E_ALL` is the one that makes this blank message appear. And even with this blank message on the app interface, the error log is still empty.

**Comment by L30Bola** [comment]
So, I did a full build of the application image, instead of use the one provided by the teampass/teampass:latest, found on the Docker Hub. The differences being:
  + Used the Alpine image as base, instead the CentOS one;
  + Installed the latest PHP 5 version available: 5.6.31, provided by WebTatic Repos: https://webtatic.com/projects/yum-repository/
  + Instead oh Nginx, I used Apache, just for the sake of testing;

After the "installation" process, I cloned the TeamPass repo for, the latest version and tested. All seem working now.

Also, on my last reply on this issue, I said that the ```error_reporting = E_ALL``` was the the PHP variable which made the blank dialog appear. But I was wrong. After reading the PHP configuration, I found out that it's the ```display_errors = 1```. 
I still didn't find the problem on the official TeamPass Docker Image problem. I'll try to find the error and report here. :)

**Comment by nilsteampassnet** [comment]
Thank you for all the tests you did here.
I don't have much knowledge of Docker. My test consists here to install it and check that data is correct.
And this was the case but I'm loading it on Win10 machine.

**Comment by L30Bola** [comment]
Does TeamPass works with PHP 7.1? If yes, which PHP version do you recommend to use with TeamPass? 5, 7 or 7.1?

**Comment by L30Bola** [comment]
I tested with the latest version of PHP 5 (5.6.31), PHP 7.0 (7.0.23) and PHP 7.1 (7.1.9). They all worked well, without this bug. I tested them all on a CentOS Docker container. I'm not sure about this, but I'm guessing the problem is inside that Docker image that is being used as the base for the current TeamPass Docker image. Maybe some missing PHP dependency/module. I didn't make all the tests to be 100% sure about it though.
